proto/initia/tx/v1/tx.proto

```
@@ -0,0 +1,11 @@
+syntax = "proto3";
+package initia.tx.v1;
+
+import "cosmos_proto/cosmos.proto";
```

Buf (1.66.0)

[error] import "cosmos_proto/cosmos.proto": file does not exist

(COMPILE)